rpm/SPECS/pepProxy.spec

```diff
@@ -90,18 +90,6 @@ echo "[INFO] Configuring application"
     setfacl -d -m g::rwx %{_pepProxy_log_dir}
```

Comment:
Maybe documentation should be reviewed, e.g. it seems that the following text is no longer valid after implementing this change:

"If the PEP Proxy is deployed in a machine with an installed Context Broker service, the PEP Proxy will automatically change the Context Broker port to the 10026 and install itself on the port where the Context Broker was listening, so no further configuration should be needed for the connectivity."

Probably there are more documentation tokens to be adjusted (I haven't checked it in long).